.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!-- If you are submitting a dataset or a model for the model registry please use the corresponding checklists below otherwise feel free to remove them. -->
```

Comment:
Nit: shorter sentences.
```suggestion
<!-- If you are submitting a dataset or a model for the model registry, please use the corresponding checklists below. Otherwise, feel free to remove them. -->
```